### PR TITLE
Add feature info with more extensive feature data.

### PR DESCRIPTION
**Pull request by tbitonti**
Testing update.  Need to make more information from the feature files available to feature resolution FAT and unit tests.

### CODE REVIEW

**Comment by tbitonti** [comment]
#build

**Comment by tbitonti** [comment]
#build

**Review by jimblye — APPROVED** [review]

File: dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/util/FeatureInfo.java [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+        if (versionIndex != -1) {
+            return featureName.substring(0, versionIndex);
+        } else {
+            return featureName;
+        }
+    }
+
+    public static String getVersion(String featureName) {
+        int versionIndex = featureName.lastIndexOf('-');
+        if (versionIndex != -1) {
```
Comment (rsherget) [inline dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/util/FeatureInfo.java:526]: May want to import `org.osgi.framework.Version` and use `Version.parseVersion()`

**Review by hlhoots — APPROVED** [review]
LGTM me pending a build.

**Review by rsherget — APPROVED** [review]
LGTM

**Comment by tbitonti** [comment]
Delivered by commit 61263fe.

**Pull request closed without merging**